src/views/templates/utils/virtualMachineTemplatesRowFilter.ts

```diff
@@ -0,0 +1,37 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getTemplateProviderName } from '@kubevirt-utils/resources/template/utils';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+export const templateProviders = [
+  {
+    id: 'KubeVirt',
+    title: 'KubeVirt',
+  },
+  {
+    id: 'User',
+    title: 'User',
+  },
+];
+
+const includeFilter = (compareData: FilterValue, compareString: string) => {
+  return (
+    compareData.selected?.length === 0 ||
+    compareData.selected?.includes(compareString) ||
+    !compareData?.all?.find((s: string) => s === compareString)
+  );
+};
+
+export const useFilters = (): RowFilter[] => {
```

Comment:
since this is a hook, it will be best if you transfer this into a separate file, under hooks directory IMO